layers/+frameworks/ruby-on-rails/packages.el

```diff
@@ -44,6 +44,8 @@
         "ffv" 'projectile-rails-find-view
         "ffy" 'projectile-rails-find-layout
         "ff@" 'projectile-rails-find-mailer
+        "ffw" 'projectile-rails-find-webpack
+        "ffb" 'projectile-rails-find-job
```

Comment:
This should be ordered alphabetically under `"ffa" 'projectile-rails-find-locale`.

Reply:
Done.

Reply:
You should do it in the documentation as well.

Reply:
Double done.